python/dgl/graphbolt/impl/neighbor_sampler.py

```diff
@@ -566,8 +566,8 @@ class LayerNeighborSampler(NeighborSamplerImpl):
         original_edge_ids=None,
         original_column_node_ids=tensor([0, 1, 5, 2]),
     )]
-    >>> next(iter(subgraph_sampler)).compacted_node_pairs
-    (tensor([0]), tensor([1]))
+    >>> next(iter(subgraph_sampler)).compacted_seeds
+    tensor([[0, 1], [0, 2], [0, 3]])
```

Comment:
Why the value changed? Should it just be tensor([[0, 1])?

Reply:
In the past, `compacted_node_pairs` only has positive node pairs. And `negtive_node_pairs` saved the negtive ones. But now, we save the both in the `compacted_seeds`, so the value changes.

Reply:
If that's the case, can you output label as well to clarify. SImilar to L46 python/dgl/graphbolt/impl/uniform_negative_sampler.py